cmd/genji/shell/shell.go

```diff
@@ -184,6 +184,16 @@ func (sh *Shell) execute(in string) {
 	}
 }
 
+// Query Error return an error if we press return with no input, or ended by whitespace
+func (sh *Shell) QueryError(in string) error {
```

Comment:
The name of this function doesn't seem to have anything to do with the content of the function, does it?